docs/docs/reference/crd-reference/app.md

```diff
@@ -31,6 +31,7 @@ spec:
 ```
 
 ## Fields
+<!-- markdownlint-disable MD007 -->
```

Comment:
you always need re-enable the rule  at the end of the section you want to ignore it for

Reply:
fixed

Reply:
fixed